plugins/parsers/csv/parser.go

```diff
@@ -96,49 +96,68 @@ func (p *Parser) compile(r io.Reader) *csv.Reader {
 
 func (p *Parser) Parse(buf []byte) ([]telegraf.Metric, error) {
 	r := bytes.NewReader(buf)
+	return parseCSV(p, r)
+}
+
+// ParseLine does not use any information in header and assumes DataColumns is set
+// it will also not skip any rows
+func (p *Parser) ParseLine(line string) (telegraf.Metric, error) {
+	r := bytes.NewReader([]byte(line))
+	metrics, err := parseCSV(p, r)
+	if err != nil {
+		return nil, err
+	}
+	if len(metrics) == 1 {
+		return metrics[0], nil
+	} else if len(metrics) > 1 {
```

Comment:
Sorry my bad, the linter will probably complain on this, just do
```suggestion
	}
	if len(metrics) > 1 {
```